{
  "totalNumberSkippingTests": 62,
  "step-durations": {
    "CheckoutBuild": 0,
    "": 0,
    "ComputeClasspath": 2,
    "CloneRepository": 28,
    "ResolveDependency": 13,
    "TestProject": 656,
    "BuildProject": 336,
    "PushIncriminatedBuild": 6,
    "SquashRepository": 7
  },
  "nbLibraries": 209,
  "workspace": "./workspace",
  "totalNumberErroringTests": 0,
  "repo": "druid-io/druid",
  "bugCommit": "https://github.com/druid-io/druid/compare/db656c5a88d3...5b69f2eff2f2",
  "failingModule": "/root/workspace/druid-io/druid/223380340/server",
  "buildid": 223380340,
  "totalNumberFailingTests": 4,
  "error-types": [
    "java.lang.AssertionError"
  ],
  "totalNumberRunningTests": 27367,
  "failing-test-cases": [
    {
      "className": "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest",
      "failingMethods": [
        "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest#testRetryAllFail",
        "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest#testRetrySuccessAtSecondLocation"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "isError": false
        }
      ],
      "nbFailures": 2,
      "nbErrors": 0
    },
    {
      "className": "io.druid.segment.loading.LocalDataSegmentPusherTest",
      "failingMethods": [
        "io.druid.segment.loading.LocalDataSegmentPusherTest#testPushCannotCreateDirectory"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Expected test to throw (an instance of java.io.IOException and exception with message a string containing \"Unable to create directory\")",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    },
    {
      "className": "io.druid.query.lookup.LookupSnapshotTakerTest",
      "failingMethods": [
        "io.druid.query.lookup.LookupSnapshotTakerTest#testIOExceptionDuringLookupPersist"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Expected exception: io.druid.java.util.common.ISE",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    }
  ],
  "bugRepo": "druid-io/druid"
}